apps/ci/ci-pending-sql.sh

```diff
@@ -40,16 +48,15 @@ function import() {
     # Get latest SQL file applied to this database, today. This could be empty.
     LATEST_UPDATE_TODAY="$(find "$UPDATES_DIR" -iname "$TODAY*.sql" | sort -h | tail -n 1)"
 
-    for entry in "$PENDING_PATH"/*.sql
-    do
+    for entry in "$PENDING_PATH"/*.sql; do
         if [[ -f "$entry" ]]; then
             INDEX="$(get_next_index "$LATEST_UPDATE_TODAY")"
             OUTPUT_FILE="${UPDATES_DIR}/${TODAY}_${INDEX}.sql"
 
             # ensure a note is added as a header comment
-            echo "-- DB update $(basename "$LATEST_UPDATE" .sql) -> $(basename "$OUTPUT_FILE" .sql)" > "$OUTPUT_FILE"
+            echo "-- DB update $(basename "$LATEST_UPDATE" .sql) -> $(basename "$OUTPUT_FILE" .sql)" >"$OUTPUT_FILE"
             # fill in the SQL contents under that
-            cat "$entry" >> "$OUTPUT_FILE"
+            cat "$entry" >>"$OUTPUT_FILE"
```

Comment:
the editor I use autoformats on save, and I accidentally let this slip through. I don't think it really matters for these specific changes since these are small and don't really hurt readability, but we can also revert these if needed.